Fixes the following error for SH4 native binutils

/scratch/oe/work/sh4-oe-linux-uclibc/binutils-git-r13.2+gitr0+6ef128735ce52ee2acf8cfd80f6d72003dd39ab7/git/gas/config/obj-elf.c:744: error: declaration of 'clone' shadows a global declaration/scratch/oe/sysroots/sh4-oe-linux-uclibc/usr/include/bits/sched.h:74: error: shadowed declaration is here/scratch/oe/work/sh4-oe-linux-uclibc/binutils-git-r13.2+gitr0+6ef128735ce52ee2acf8cfd80f6d72003dd39ab7/git/gas/config/obj-elf.c: In function 'obj_elf_section':/scratch/oe/work/sh4-oe-linux-uclibc/binutils-git-r13.2+gitr0+6ef128735ce52ee2acf8cfd80f6d72003dd39ab7/git/gas/config/obj-elf.c:981: error: declaration of 'clone' shadows a global declaration/scratch/oe/sysroots/sh4-oe-linux-uclibc/usr/include/bits/sched.h:74: error: shadowed declaration is heremake[4]: *** [obj-elf.o] Error 1

-Khem

Index: git/gas/config/obj-elf.c
===================================================================
--- git.orig/gas/config/obj-elf.c
+++ git/gas/config/obj-elf.c
@@ -741,10 +741,10 @@ obj_elf_change_section (const char *name
 }
 
 static bfd_vma
-obj_elf_parse_section_letters (char *str, size_t len, bfd_boolean *clone)
+obj_elf_parse_section_letters (char *str, size_t len, bfd_boolean *duplicate)
 {
   bfd_vma attr = 0;
-  *clone = FALSE;
+  *duplicate = FALSE;
 
   while (len > 0)
     {
@@ -775,7 +775,7 @@ obj_elf_parse_section_letters (char *str
 	  attr |= SHF_TLS;
 	  break;
 	case '?':
-	  *clone = TRUE;
+	  *duplicate = TRUE;
 	  break;
 	/* Compatibility.  */
 	case 'm':
@@ -978,7 +978,7 @@ obj_elf_section (int push)
 
       if (*input_line_pointer == '"')
 	{
-	  bfd_boolean clone;
+	  bfd_boolean duplicate;
 
 	  beg = demand_copy_C_string (&dummy);
 	  if (beg == NULL)
@@ -986,7 +986,7 @@ obj_elf_section (int push)
 	      ignore_rest_of_line ();
 	      return;
 	    }
-	  attr |= obj_elf_parse_section_letters (beg, strlen (beg), &clone);
+	  attr |= obj_elf_parse_section_letters (beg, strlen (beg), &duplicate);
 
 	  SKIP_WHITESPACE ();
 	  if (*input_line_pointer == ',')
@@ -1038,10 +1038,10 @@ obj_elf_section (int push)
 	      attr &= ~SHF_MERGE;
 	    }
 
-	  if ((attr & SHF_GROUP) != 0 && clone)
+	  if ((attr & SHF_GROUP) != 0 && duplicate)
 	    {
 	      as_warn (_("? section flag ignored with G present"));
-	      clone = FALSE;
+	      duplicate = FALSE;
 	    }
 	  if ((attr & SHF_GROUP) != 0 && *input_line_pointer == ',')
 	    {
@@ -1063,7 +1063,7 @@ obj_elf_section (int push)
 	      attr &= ~SHF_GROUP;
 	    }
 
-	  if (clone)
+	  if (duplicate)
 	    {
 	      const char *now_group = elf_group_name (now_seg);
 	      if (now_group != NULL)
